server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -132,9 +132,9 @@ For more detail about how caches can be configured see <<tuningcache>>.
 [[communication]]
 ==== Communication details
 
-{project_name} uses multiple, separate clusters of Infinispan caches. Every {project_name} node is in the cluster with the other {project_name} nodes in same data center, but not with the {project_name} nodes in different data centers. A {project_name} node does not communicate directly with the {project_name} nodes from different data centers. {project_name} nodes use external JDG (actually {jdgserver_name} servers) for communication across data centers. This is done using the link:https://infinispan.org/docs/8.2.x/user_guide/user_guide.html#using_hot_rod_server[Infinispan HotRod protocol].
+{project_name} uses multiple, separate clusters of Infinispan caches. Every {project_name} node is in the cluster with the other {project_name} nodes in same data center, but not with the {project_name} nodes in different data centers. A {project_name} node does not communicate directly with the {project_name} nodes from different data centers. {project_name} nodes use external JDG (actually {jdgserver_name} servers) for communication across data centers. This is done using the link:https://infinispan.org/docs/stable/titles/hotrod_java/hotrod_java.html[Infinispan HotRod protocol].
```

Comment:
Change "HotRod protocol" to "Hot Rod protocol"

Reply:
In this file there are many other occurrences of HotRod, so if I change this I will also have to change all occurrences, which looks like a deviation from the purpose of this PR. If you think it is fine to include all these changes I can do it, just wanted to bring this up.

Reply:
Agree, in that case let's fix it in another doc PR. This PR is about fixing links, and that is done.

Reply:
I have already changed them in my PR for Data Grid 8.1/Infinispan 11.0.3 anyway.